scripts/entrypoint.sh

```diff
@@ -28,4 +28,4 @@ if $LOAD_FIXTURES == "true"; then
     python manage.py loaddata /tmp/fake_agagd_data.json
 fi
 
-uwsgi --http-socket 0.0.0.0:3031 --module agagd.wsgi --static-map /static=/tmp/static/ --static-map /media=/srv/media
+uwsgi --http-socket 0.0.0.0:3031 --module agagd.wsgi --static-map /static=/tmp/static/ --static-map /media=/srv/media --touch-reload /srv/agagd/uwsgi-reload
```

Comment:
It might be nice to have some notes about `--touch-reload` and how it may be used. Since the entrypoint is the same for production, are there any consequences to not having the file for `--touch-reload`?

Reply:
It seems to run fine when it's not there.  I'm not sure if there are negative side effects for its absence.

I guess strictly speaking, if it's there for production, it'd cause some downtime when it's touched.

Good point on adding a note.  I can do that.

Reply:
I'll start with a comment on `touch-reload` in this file.

Any thoughts on doing the `docker-compose up` via `make` to support the dev case?

Reply:
What do you mean by make?

Reply:
I think having a dev case may help avoid some of the unknowns with `--touch-reload`. What do you mean by "`docker-compose up` via `make`"?

Reply:
`make` as in [`Makefile`](https://en.wikipedia.org/wiki/Makefile).

Sorry for not being clear - I meant having a makefile that runs the following:
* watcher on `*.py` and touches `uwsgi-reload`
* `docker-compose up`

Then running the server in a dev environment would be something like `make start`.  It should be easy to add to this MR and it might clear up some confusion.  I was just being lazy (and I ran out of time).

Reply:
Thanks. 

I wonder if it would be beneficial to modify the startup order here: https://docs.docker.com/compose/startup-order/ and change the command option in docker-compose for the dev case.